code/src/java/pcgen/util/chooser/RandomChooser.java

```diff
@@ -52,33 +51,14 @@
 	private int totalSelectionsAvailable = 1;
 
 	private boolean pickAll = false;
-	
-	/**
-	 * Chooser constructor.
-	 */
-	public RandomChooser()
-	{
-		//Empty constructor
-	}
-
-	/**
-	 * Sets the AllowsDups attribute of the Chooser object
-	 *
-	 * @param aBool  The new AllowsDups value
-	 */
-    @Override
-	public void setAllowsDups(boolean aBool)
-	{
-		theAllowDuplicatesFlag = aBool;
-	}
```

Comment:
I'm pretty sure that this setter has a purpose, even if it isn't used. Please don't remove it.